comfyui_blender/utils.py

```diff
@@ -16,7 +16,7 @@ def download_file(filename, subfolder):
 
     # Download the file data from the ComfyUI server
     # Add a random parameter to avoid caching issues
-    params = {"filename": filename, "subfolder": subfolder, "type": "output", "rand": random.random()}
+    params = {"filename": filename, "subfolder": subfolder, "type": output_type, "rand": random.random()}
```

Comment:
Rename argument from `output_type` to `type` to align with the websocket message / other arguments.

```suggestion
    params = {"filename": filename, "subfolder": subfolder, "type": type, "rand": random.random()}
```